docs/user-guide/api-mediation/api-mediation-onboard-an-existing-rest-api-service-without-code-changes.md

```diff
@@ -231,21 +269,21 @@ The API Mediation Layer prints following messages to its log when the API defini
 
 ## (Optional) Reload the services definition after the update when the API Mediation Layer is already started
 
-The following procedure enables you to refresh the API definitions after you change the definitions while the API Mediation Layer is already started.
+The following procedure enables you to refresh the API definitions after you change the definitions whin the API Mediation Layer is already running.
 
 **Follow these steps:**
 
 1. Use a REST API client to issue a POST request to the Discovery Service (port 10011):
     
         http://localhost:10011/discovery/api/v1/staticApi
 
-    Discovery Services requires authentication. In the case of the API Mediation Layer running on your local machine, it is `eureka` as the user ID and `password` as the password.
+    Discovery Services requires authentication. If the API Mediation Layer running on your local machine, the user ID is `eureka`, and the password is `password`.
 
-    This is an example that uses the [HTTPie command-line HTTP client](https://httpie.org):
+    This example uses the [HTTPie command-line HTTP client](https://httpie.org):
    
         http -j -a eureka:password POST http://localhost:10011/discovery/api/v1/staticApi
 
-2. Check if your updated definition is effective
+2. Check if your updated definition is effective.
```

Comment:
How do you check if the updated definition is effective?

Reply:
In the same way as at the end of previous sections:  

> Go to the following URL to reach the API Gateway...

I did not want to repeat it again.